frontend/test/pytest/test_transform.py

```diff
@@ -121,44 +166,896 @@ def qfunc(data, x, weights):
     assert expected_shape == observed_shape
 
 
-def test_split_non_commuting(backend):
-    """Test split non commuting"""
+@pytest.mark.xfail(reason="JIT and QJIT give different results.")
```

Comment:
How about qjit and plain pennylane?

Reply:
This turned out to be an error in my test, after fixing it the test passes 🥳